web/src/OverviewActionBar.tsx

```diff
@@ -659,6 +660,88 @@ export function OverviewWidgets(props: { buttons?: UIButton[] }) {
     </WidgetRoot>
   )
 }
+const EnableDisableResourceRoot = styled(ButtonRoot)<{
+  confirmingDisable: boolean
+}>`
+  ${(props) =>
+    props.confirmingDisable &&
+    `
+    color: ${Color.white};
+    background-color: ${Color.red};
+
+    &:hover {
+      color: ${Color.white};
+      background-color: ${Color.red};
+    }
+  `};
+  margin-left: auto;
```

Comment:
(design nitpick) i'm not a huge fan of the layout this margin creates, since it acts a little bit like a `justify-content: space-around`. from the designs, it looks like both the clear logs menu and disable button are right-aligned, so i think you could get away with `margin-left: ${SizeUnit(1 / 2)}` for now.